tests/instructions/extcodehash_test.py

```diff
@@ -0,0 +1,44 @@
+from mythril.disassembler.disassembly import Disassembly
+from mythril.laser.ethereum.state.environment import Environment
+from mythril.laser.ethereum.state.account import Account
+from mythril.laser.ethereum.state.machine_state import MachineState
+from mythril.laser.ethereum.state.global_state import GlobalState
+from mythril.laser.ethereum.state.world_state import WorldState
+from mythril.laser.ethereum.instructions import Instruction
+from mythril.laser.ethereum.transaction.transaction_models import MessageCallTransaction
+
+from mythril.support.support_utils import get_code_hash
+
+from mythril.laser.smt import symbol_factory
+
+
+def test_extcodehash_concrete():
```

Comment:
I think we can make it into 3 separate tests, Because if we have it combined then we may not execute all the tests,(if the top one fails then the ones below aren't executed)

Reply:
Ah I noticed that. Good point. Thanks!